README.md

```diff
@@ -39,14 +39,51 @@ A `validate.sh` script is provided for convenience.
 
 Note: JSON Schemas only capture the NGSI simplified representation (`options=keyValues`)
 
+
+
+
 ## How to contribute
 
-Contributions should come in the form of pull requests. 
+Contributions should come in the form of pull requests.
+
+New data models should include:
+- A data model description based on the [data model template](https://github.com/Fiware/dataModels/blob/master/datamodel_template.md), e.g. [spec.md](https://github.com/Fiware/dataModels/blob/master/Weather/WeatherObserved/doc/spec.md) (model specifications should be stored in the doc folder of the data model)
```

Comment:
I think we are missing a clear example (i.e, not making explicit) the expected layout of the files created in a new PR for a new model. Maybe we should give a tree-like view will help to get right both the structure and the file names.

E.g

    - NewModel/
        - doc/
            - spec.md
        - README.md
        - schema.json
        - example.json

Same thing for the tests.